lineman/app/components/profile_page/profile_page_controller.coffee

```diff
@@ -1,17 +1,35 @@
-angular.module('loomioApp').controller 'ProfilePageController', ($rootScope, CurrentUser, Records, FlashService, $location) ->
-  @user = CurrentUser.clone()
+angular.module('loomioApp').controller 'ProfilePageController', ($rootScope, Records, FlashService, $location, AbilityService, ModalService, ChangePictureForm, ChangePasswordForm, DeactivateUserForm) ->
+  @init = ->
+    @user = Records.users.find(window.Loomio.currentUserId)
```

Comment:
I've had to refind the current user after profile page update to get the profile picture to rebind correctly. 

I suspect it's something with the new angular router, but then again I blame the hole in the ozone layer over New Zealand on the new angular router too.